.github/workflows/uiTests.yml

```diff
@@ -0,0 +1,48 @@
+name: Run UI Tests
+
+on:
+  schedule:
+    - cron: '0 0 * * *' 
+  
+  pull_request:
+    branches:
+      - main 
+
+jobs: 
+  uiTests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 20.14.0
+    
+    - name: Install dependencies
+      run: npm install
+  
+    - name: Install Google Chrome
```

Comment:
can't you just run the ui tests in headless mode?

Reply:
They are but had to install specific version of chrome compatible with chromedriver